filesmanager/tasks.py

```diff
@@ -0,0 +1,68 @@
+import tempfile
+import zipfile
+
+from celery import shared_task
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.core.files.storage import default_storage
+from xmodule.contentstore.django import contentstore
+
+try:
+    from opaque_keys.edx.keys import AssetKey
+except ImportError:
+    AssetKey = None
```

Comment:
we can add [openedx/opaque-keys](https://github.com/openedx/opaque-keys) as a requirement